tests/framework/db/ActiveQueryTest.php

```diff
@@ -129,6 +129,24 @@ public function testInnerJoinWith()
         ], $result->joinWith);
     }
 
+    /**
+     * @todo: tests for the regex inside getQueryTableName
+     */
+    public function testGetQueryTableName_from_not_set()
```

Comment:
an alternative could be testGetQueryTableName_fromNotSet(), its a matter of taste and the depth in which you are testing. If you have 20 different test methods with similarities, the underscores help significantly with an easy comparison